### PR TITLE
Outdated: Make compatible with Dekode Starter Theme

### PR DESCRIPTION
**Pull request by pdewouters**
- Replaces `dekode-theme` with `dekode-starter-theme`
- Modifies the webpack config so that developers can use a block library plugin for all custom blocks (taken from an existing working project)
- CSS linting does not use SCSS preset anymore
- Include npm packages from the theme in the main `package,json`

### CODE REVIEW

**Review by walbo — APPROVED** [review]

**Comment by stian-overasen** [comment]
Do we want a full out-of-sync copy of the Starter Theme in this repo?

**Comment by walbo** [comment]
I was planning on setting up a autosync from the theme repo after this was merged. In the autosync we can remove all unwanted files.

File: packages/mu-plugins/project-base-block-library/inc/.gitkeep
Comment (stian-overasen) [inline packages/mu-plugins/project-base-block-library/inc/.gitkeep:1]: How about just `block-library`?

**Comment by Sisanu** [comment]
Shouldn't this tooling also include something about external plugins? I know we are using at least one (Gravity Forms) in all the projects.

**Comment by pdewouters** [comment]
@Sisanu can you be more specific? The goal here is to just make project base work with the new starter theme.

**Comment by stian-overasen** [comment]
> Shouldn't this tooling also include something about external plugins? I know we are using at least one (Gravity Forms) in all the projects.

Adding Gravity Forms as a default plugin to Project Base would be a different PR.

**Comment by Sisanu** [comment]
> @Sisanu can you be more specific? The goal here is to just make project base work with the new starter theme.

From my perspective, adding the webpack configs or what else is required related to using external plugins should also be part of the project base. Because that needs to be tested that works properly in a real scenario, with the rest of the packages and assets handling we are using.

**Comment by olethomas** [comment]
AXE: https://dekode.no

**Pull request closed without merging**